dockerfile_functions.sh

```diff
@@ -76,6 +76,25 @@ print_ubi-minimal_ver() {
 	EOI
 }
 
+print_centos_ver() {
+	os_version="7"
+
+	cat >> $1 <<-EOI
+	FROM centos:${os_version}
+
+	EOI
+}
+
+print_clefos_ver() {
+	os_version="7"
```

Comment:
Same here

Reply:
>A ClefOS 8 build is underway but it will be some time before it’s available.
It will not be possible until ClefOS is done building. Here are the available tags: https://hub.docker.com/_/clefos?tab=tags